docs/ops/detection/ExperimentalDetectronGenerateProposalsSingleImage_9.md

```diff
@@ -0,0 +1,126 @@
+# ExperimentalDetectronGenerateProposalsSingleImage {#openvino_docs_ops_detection_ExperimentalDetectronGenerateProposalsSingleImage_9}
```

Comment:
We need to make this operation aligned with TensorFlow, Caffe2, and PDPD operations:
- https://www.tensorflow.org/api_docs/python/tf/image/generate_bounding_box_proposals
- https://caffe2.ai/docs/operators-catalogue.html#generateproposals
- https://github.com/PaddlePaddle/PaddleDetection/blob/3c78716d30945e89129c1839453932482f4a6974/ppdet/modeling/ops.py#L1424